projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/GeneratedRoute.java

```diff
@@ -118,8 +120,8 @@ public GeneratedRoute(
     _discard = discard;
     _generationPolicy = generationPolicy;
     _metric = metric;
+    _nextHopIp = firstNonNull(nextHopIp, Route.UNSET_ROUTE_NEXT_HOP_IP);
     _nextHopInterface = MoreObjects.firstNonNull(nextHopInterface, Route.UNSET_NEXT_HOP_INTERFACE);
```

Comment:
on 123 you use `firstNonNull` ; on 124 you use `MoreObjects.firstNonNull` -- pick one and be consistent throughout?

I prefer just `firstNonNull` – it's cleaner.